ksql-parser/src/main/java/io/confluent/ksql/parser/KsqlParser.java

```diff
@@ -61,11 +62,11 @@ public SingleStatementContext getStatement() {
     }
   }
 
-  public static final class PreparedStatement {
+  public static final class PreparedStatement<T extends Statement> {
```

Comment:
I've made `PreparedStatement` generic so that handler methods in `KsqlResource` and else where can be type safe.